recipes-extended/dpdk/dpdk.inc

```diff
@@ -82,59 +86,74 @@ do_install () {
     oe_runmake O=${RTE_OUTPUT} T= install-sdk DESTDIR=${D}
 
     # install examples without openssl enabled
-    install -m 0755 -d ${D}/${datadir}/dpdk/examples
+    install -m 0755 -d ${D}${datadir}/examples
```

Comment:
Generally, the structure is general -> specific, read:
${datadir}/examples/dpdk
${datadir}/doc/dpdk
...

not
${datadir}/dpdk/$(own structure)

The latter one is discouraged.

Reply:
Year, agree with you. dpdk default to install to ${datadir}/dpdk/, just follow it.

Reply:
Is /usr/share just wrong to begin with? /usr/share is supposed to be for architecture-independent data. Wouldn't /opt be more appropriate?

https://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.pdf

Reply:
Then you would have to install everything qoriq related to /opt. I had a longer discussion with @rpurdie and @rossburton wrt. /opt and Yocto etc. - and it will need some fixes before it's sanely usable.

Also - dpdk isn't architecture dependent - it's possible to have different architectures using dpaa* and dpdk.